superset/utils/webdriver.py

```diff
@@ -97,6 +104,7 @@ def get_screenshot(
         self, url: str, element_name: str, user: "User",
     ) -> Optional[bytes]:
 
+        url = f"{url}?standalone={DashboardStandaloneMode.REPORT}"
```

Comment:
This will break if we change the URL in `utils/screenshots.py` to include other parameters, it would be safer to parse the URL and append the new parameter programmatically instead of doing string manipulation.

An easy way to do that is using the `requests` library:

```suggestion
        from requests.models import PreparedRequest
        params = {"standalone": DashboardStandaloneMode.REPORT.value}
        req = PreparedRequest()
        req.prepare_url(url, params)
        url = req.url
```

Or we can use `urllib.parse`, but it's more code: https://stackoverflow.com/a/25580545/807118